src/settings-ui/Settings.UI.Library/ZoomItProperties.cs

```diff
@@ -78,7 +78,7 @@ public ZoomItProperties()
 
         public BoolProperty ShowTrayIcon { get; set; }
 
-        public BoolProperty AnimnateZoom { get; set; }
+        public BoolProperty AnimateZoom { get; set; }
```

Comment:
I'm not sure how public a public is, but I hope it doesn't mind being renamed...